Src/Fare/RandomGenerator.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+
+namespace Fare
+{
+    public static class RandomGenerator
```

Comment:
What's the advantage of this PRNG compared to others? E.g. SplitMix, [PCG](https://www.pcg-random.org/index.html), etc.

Reply:
Oh no idea,didn't give it much thought and this was just the natural implementation :-)

Reply:
What do you mean by *natural*?